README.md

```diff
@@ -323,14 +323,14 @@ Current release info
 Installing rust-activation
 ==========================
 
-Installing `rust-activation` from the `conda-forge` channel can be achieved by adding `conda-forge` to your channels with:
+Installing `rust-activation` from the `conda-forge/label/rust_dev` channel can be achieved by adding `conda-forge/label/rust_dev` to your channels with:
```

Comment:
eee, this looks weird. but then, not so many folk knowingly _install_ this package...

Reply:
Yeah, that's okay